config/locales/en.yml

```diff
@@ -408,6 +412,12 @@ en:
           attributes:
             terms_accepted:
               accepted: "You must accept the terms and conditions to continue"
+        publish/authentication/magic_link_form:
+          attributes:
+            email:
+              blank: "Enter an email adddress"
+              invalid: "Enter a valid email address"
+              does_not_exist: "We cannot find an account with that email address"
```

Comment:
This looks like a bad idea, as thisis a means to validated an email account holder access.

https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html#authentication-responses

Reply:
Re-review? 🙏

Reply:
Just needs to be removed